### PR TITLE
fix ts-client

### PR DESCRIPTION
**Pull request by elshan-eth**

### CODE REVIEW

File: ts-client/package.json [hunk 1]
```diff
@@ -2,19 +2,23 @@
   "name": "@fluencelabs/deal-aurora",
   "version": "0.2.22",
   "main": "dist/index.js",
+  "type": "module",
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "npm run generate-contract-types && npm run update-deployments && shx rm -rf dist && tsc -b",
-    "generate-contract-types": "typechain --target ethers-v6 '../out/**/*.json' --out-dir typechain-types",
-    "update-deployments": "rm -f -r deployments && cp -r ../deployments ./deployments",
+    "build:ts": "shx rm -rf dist && tsc -b",
```
Comment (AlcibiadesCleinias) [inline ts-client/package.json:9]: ts? seems we build dist here, package?

File: ts-client/.gitignore [hunk 2]
```diff
@@ -4,6 +4,8 @@
 dist
 tmp
 /out-tsc
+typechain-types
+deployments
```
[the remark this answers is not in the document]
Comment (AlcibiadesCleinias) [reply ts-client/.gitignore:8]: to find contract address: users will go and check on npm?

what about the next deploy on the public network? how others will know about previous deployment?

File: ts-client/package.json (hunk 1 above)
Comment (elshan-eth) [reply ts-client/package.json:9]: This means that this is a building of ts code. `build:dist` is a more obscure name.

File: ts-client/.gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (elshan-eth) [reply ts-client/.gitignore:8]: deployments are in the root directory. In ts-client, it appears when we build because we need it in npm.

**Comment by AlcibiadesCleinias** [comment]
I merged into https://github.com/fluencelabs/deal/pull/146

**Pull request closed without merging**